File: python/secret.py
import random

runes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"

def rand_seq(n):
    b = []
    for i in range(n):
        b.append(random.choice(runes))
    return "".join(b)

def generate_secret_key():
    return rand_seq(32)

File: go/secret.go
package secureput

import (
	"math/rand"
	"time"
)

var runes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")

// https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-go/22892986#22892986
func RandSeq(n int) string {
	b := make([]rune, n)
	for i := range b {
		b[i] = runes[rand.Intn(len(runes))]
	}
	return string(b)
}

func GenerateSecretKey() string {
	rand.Seed(time.Now().UnixNano())
	return RandSeq(32)
}
